transform/mattermost-analytics/macros/rudderstack/_rudderstack_schema.yml

```diff
@@ -22,3 +22,17 @@ macros:
         type: list
         description: The list of columns to keep.
 
+  - name: get_event_relations
+    description: |
+      Returns a list of relations from the given schema that might be Rudderstack events. Offers functionality for 
```

Comment:
maybe mention that there relations are tables (and not other types of schema elts like views)